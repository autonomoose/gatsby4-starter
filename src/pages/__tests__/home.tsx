import React from "react";
import { render } from "@testing-library/react";

import HomePage from "../home";

const mockEnqueue = jest.fn();
jest.mock('notistack', () => ({
  ...jest.requireActual('notistack'),
  useSnackbar: () => {
    return {
      enqueueSnackbar: mockEnqueue
    };
  }
}));

describe("HomePage", () => {
  it("renders snapshot correctly", () => {
    const {container} = render(<HomePage />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
