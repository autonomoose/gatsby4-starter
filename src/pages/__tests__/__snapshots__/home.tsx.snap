// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders snapshot correctly 1`] = `
<div
  class="MuiBox-root css-1yp4ln"
>
  <div
    class="MuiBox-root css-pntgf1"
  >
    <h1
      id="site-name"
    >
      Starter Home
    </h1>
  </div>
</div>
`;
